cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,16 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => cy.visit('https://conduit.mate.academy/user/login'));
```

Comment:
The `visit` command should use a relative URL or a configuration variable for the base URL. Hardcoding the full URL can make the tests less flexible and harder to run in different environments.